client/e2e-test/script-test.js

```diff
@@ -158,9 +171,19 @@ async function test(page, shouldLogin) {
   console.log(`Dashboard ${datasetName} was successfully created.\n`);
 }
 
-
-let browser;
-let page;
+const getSourcemapsScript = () => new Promise((resolve) => {
+  https.get('https://raw.githubusercontent.com/akvo/akvo-lumen/develop/client/vendor/sourcemapped-stacktrace.js', (res) => {
+    res.setEncoding('utf8');
+    let body = '';
+    res.on('data', (data) => {
+      body += data;
+    });
+    res.on('end', () => {
+      sourcemapScript = body;
+      resolve(body);
```

Comment:
It's a small point, but perhaps we should change this to `resolve()` just to be a bit clearer that we're storing the value using the parent scope rather than the resolve value?